pkg/sbom/io/decode.go

```diff
@@ -256,6 +256,13 @@ func (m *Decoder) pkgName(pkg *ftypes.Package, c *core.Component) string {
 		return pkg.Name
 	}
 
+	// Bitnami used different pkg.Name and the name from PURL.
```

Comment:
Since @fmulero mentioned it would be rotated in 6-9 months, how about adding a comment like this?

```suggestion
	// TODO(backward compatibility): Remove after 03/2025
	// Bitnami used different pkg.Name and the name from PURL.
```

Reply:
Good idea! Added in https://github.com/aquasecurity/trivy/pull/6982/commits/7e76a4b78a3b56f782f485727c544af283f7663f